config/config.yaml

```diff
@@ -1,6 +1,17 @@
-project_code: "sage-bionetworks"
-profile: {{ var.profile | default("default") }}
-region: {{ var.region | default("us-east-1") }}
-template_bucket_name: "bootstrap-awss3cloudformationbucket-114n2ojlbvj21"
-template_key_prefix: {{ environment_variable.TRAVIS_BRANCH | default("testing") }}
-admincentral_cf_bucket: "bootstrap-awss3cloudformationbucket-19qromfd235z9"
+project_code: 'sage-bionetworks'
+profile: {{ var.profile | default('default') }}
+region: {{ var.region | default('us-east-1') }}
+docker_image_tag: {{ environment_variable.DOCKER_IMAGE_TAG | default('staging-latest') }}
+domain_name: {{ environment_variable.DOMAIN_NAME | default('cri-iatlas.org') }}
+sub_domain_name: {{ environment_variable.SUB_DOMAIN_NAME | default('api') }}
+domain_name_staging: {{ environment_variable.DOMAIN_NAME_STAGING | default('cri-iatlas.org') }}
+sub_domain_name_staging: {{ environment_variable.SUB_DOMAIN_NAME_STAGING | default('api-staging') }}
+# DO NOT store the token in git.
+gitlab_reg_token: {{ environment_variable.GITLAB_REG_TOKEN | default('_NEVER_GONNA_GET_IT_') }}
```

Comment:
The GITLAB_REG_TOKEN is set in GitLab and is passed in the CI/CD. When doing the manual deploys, the default will work.